e2e/test/scenarios/embedding/embedding-questions.cy.spec.js

```diff
@@ -198,6 +223,12 @@ describe("scenarios > embedding > questions ", () => {
 
     openStaticEmbeddingModal();
 
+    modal().within(() => {
```

Comment:
Maybe let's throw this into `e2e-embedding-helpers` or a function at the bottom of this file since it's being repeated a bunch